src/main.ts

```diff
@@ -291,15 +296,30 @@ function handleDatasetChange({ currentTarget }: Event): void {
 function handleFeatureChange({ currentTarget }: Event): void {
   const value = (currentTarget as HTMLOptionElement).value;
   console.log(value);
-  canv.setFeature(value);
+  updateFeature(value);
+}
+
+function updateFeature(newFeatureName: string): void {
+  const featureData = dataset?.getFeatureData(newFeatureName);
```

Comment:
not meaning to complicate this, but would dataset.featureNames.includes(newFeatureName) work here?   It seems like we could even have a shortcut Dataset.hasFeature(name:string) to hide a bit of the details. It would be used in a couple of places...